### PR TITLE
Nahornyi lab1

### PR DESCRIPTION
**Pull request by sholmez**

### CODE REVIEW

File: Lab/Nahornyi/Lab1.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("#HelloWorld")
```
Comment (NeShkApp) [inline Lab/Nahornyi/Lab1.py]: Перероблюй.

**Review by NeShkApp — CHANGES_REQUESTED** [review]

File: Lab/Nahornyi/Lab2.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+   print("ціна зі знижкою = ", int(price) - zn)
+elif int(price) > 500 and int(price) <= 1000:
+   zn = 0.05 * int(price)
+   print("ціна зі знижкою = ", int(price) - zn)
+elif int(price) > 1000:
+   zn = 0.1 * int(price)
+   print("ціна зі знижкою = ", int(price) - zn)
+
+
+
```
Comment (sakhnenkoff) [inline Lab/Nahornyi/Lab2.py:17]: too many empty lines, please remove

File: Lab/Nahornyi/Lab2.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+aaaa = input("ВВЕДІТЬ: ")
+aaaa_len = len(aaaa)
+not_empty = False
+if(aaaa_len > 0):
+   not_empty = True
+state = aaaa if not_empty else "None"
+print (state)
```
Comment (sakhnenkoff) [inline Lab/Nahornyi/Lab2.py:24]: what state California, Montana or Texas ?? please specify

File: Lab/Nahornyi/Lab3.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+fli = []
+
+for x in li:
+    if x not in dli:
+        fli.append(x)
+print("Видалені парні номери: ", fli)
+
+for y in range(0, len(fli)):
+    fli[y] **= 2
+print("В квадраті: ", fli)
```
Comment (sakhnenkoff) [inline Lab/Nahornyi/Lab3.py:23]: in this one 🟥 or this ⬛️ ?
please specify

**Review by sakhnenkoff — CHANGES_REQUESTED** [review]

**Pull request closed without merging**